atlas-slurm

```diff
@@ -134,13 +134,18 @@ else
         logFile=
         if [ -n "$logPrefix" ]; then
             logFile=${logPrefix}.out
-        fi 
-	sleep 10
-	slurm_monitor_job "$slurmJobId" "$pollFreqSecs" "$logFile" "$monitorStyle" "$logCleanup" "$returnStdout" "$quiet"
+        fi
+        sleep 10
+        slurm_monitor_job "$slurmJobId" "$pollFreqSecs" "$logFile" "$monitorStyle" "$logCleanup" "$returnStdout" "$quiet"
         slurmExitCode=$?
 
         if [ $slurmExitCode -ne 0 ]; then
             die "Command \"$commandString\" failed" "$slurmExitCode" "$quiet"
         fi
     fi
+
+    # Generate summary statistics
+    summaryStats=$( slurm_resource_usage_summary $slurmJobId 2> /dev/null )
+    warn "\n\n==SLURM job efficiency report==\n${summaryStats}" "$quiet"
```

Comment:
Maybe check that `summaryStats` is not empty?
```suggestion
    if [[ -z "$summaryStats" ]]; then
        die "Warning: No SLURM job efficiency report generated." "$quiet"
    else
        warn "\n\n==SLURM job efficiency report==\n${summaryStats}" "$quiet"
    fi
```

Reply:
My earlier changes have outdated this diff and now I can't commit it, will commit separately.

Reply:
Done in 4972d13b11214ad88f6cb849b6f0c8348032c1d1